src/ansys/simai/core/data/training_data.py

```diff
@@ -88,6 +89,11 @@ def get_subset(self, project: Identifiable["Project"]) -> Optional[str]:
         project_id = get_id_from_identifiable(project, default=self._client._current_project)
         return self._client._api.get_training_data_subset(project_id, self.id).get("subset")
 
+    def set_subset(self, project: Identifiable["Project"], subset: SubsetEnum) -> None:
```

Comment:
1. nit: you could consider `make_subset` as a name of the method too.

2. for this function an `Args` section is needed, similar to `get_subset` that it would be consumed by the documentation. @romane-m may have suggestions on this too.